cmd/operator-sdk/run/local/local.go

```diff
@@ -40,41 +40,42 @@ import (
 	"github.com/operator-framework/operator-sdk/pkg/log/zap"
 )
 
-type runLocalArgs struct {
-	kubeconfig           string
-	watchNamespace       string
-	operatorFlags        string
-	ldFlags              string
-	enableDelve          bool
-	ansibleOperatorFlags *aoflags.AnsibleOperatorFlags
-	helmOperatorFlags    *hoflags.HelmOperatorFlags
+type RunLocalCmd struct {
+	Kubeconfig           string
+	WatchNamespace       string
+	OperatorFlags        string
+	LDFlags              string
+	EnableDelve          bool
+	AnsibleOperatorFlags *aoflags.AnsibleOperatorFlags
+	HelmOperatorFlags    *hoflags.HelmOperatorFlags
 }
 
-func (c *runLocalArgs) addToFlags(fs *pflag.FlagSet) {
-	prefix := "[local only] "
-
+func (c *RunLocalCmd) AddToFlags(fs *pflag.FlagSet) {
 	// The main.go and manager.yaml scaffolds in the new layout do not support the WATCH_NAMESPACE
 	// env var to configure the namespace that the operator watches. The default is all namespaces.
 	// So this flag is unsupported for the new layout.
 	if !kbutil.HasProjectFile() {
-		fs.StringVar(&c.watchNamespace, "watch-namespace", "",
-			prefix+"The namespace where the operator watches for changes. Set \"\" for AllNamespaces, "+
-				"set \"ns1,ns2\" for MultiNamespace")
+		fs.StringVar(&c.WatchNamespace, "watch-namespace", "",
+			"The namespaces where the operator watches for changes. "+
+				"Set this flag if your operator is not cluster-scoped")
```

Comment:
Should not change the description here of the flag and it is NOT accurate.

Reply:
Using this terminology is weird if not using OLM, which this command does not. I will change this back and make a follow up PR where we can discuss this.